services/fuzzmanager/setup-fuzzmanager.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -ex
+
+#### Install FuzzManager and its server mode requirements
+
+HOME="/home/fuzzmanager"
```

Comment:
How would you suggest integrating the two `RUN` commands into one?

Reply:
Like you had it before. ;-) You didn't necessarily need to move the script out of the `RUN` command, only adjust the path in the `setup-fuzzmanager.sh` by re-assigning `HOME` to your worker directory of the created `fuzzmanager` user which happens before, since you later do a recursive `chown` of the checked-out directory.